packages/combo-box/test/lazy-loading.common.js

```diff
@@ -1164,117 +1164,6 @@ describe('lazy loading', () => {
       });
     });
 
-    describe('undefined size', () => {
-      const ESTIMATED_SIZE = 1234;
-      const allItems = makeItems(ESTIMATED_SIZE);
-
-      it('should restore the scroll position after size update', () => {
-        const targetItemIndex = 75;
-        comboBox.dataProvider = getDataProvider(allItems);
-        comboBox.opened = true;
-        comboBox._scrollIntoView(targetItemIndex);
-        comboBox.size = 300;
-        // Verify whether the scroller not jumped to 0 pos and restored properly,
-        // having the item with 'targetItemIndex' in the bottom
-        // (exact visible items may vary depending of window size),
-        // and sometimes the 'ironList.scrollToIndex' does not point
-        // precisely to the given index, so use some margin
-        const scrollMargin = 5;
-        const expectedFirstVisibleIndex = targetItemIndex - getVisibleItemsCount(comboBox) - scrollMargin;
-        expect(getViewportItems(comboBox)[0].index).to.be.greaterThan(expectedFirstVisibleIndex);
-        expect(getViewportItems(comboBox).pop().index).to.be.lessThan(targetItemIndex + 1);
-      });
-
-      it('should reset to 0 when filter applied and filtered items size more than page size', () => {
```

Comment:
This test isn't related to the data provider at all,  so I moved it to the `internal-filtering` suite.